xbmc/video/VideoDatabase.cpp

```diff
@@ -4905,11 +4909,25 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE settings ADD VideoStream integer");
     m_pDS->exec("ALTER TABLE streamdetails ADD strVideoLanguage text");
   }
+
+  if (iVersion < 104)
+  {
+    m_pDS->exec("ALTER TABLE tvshow ADD duration INTEGER");
+
+    std::string sql = PrepareSQL("SELECT idShow, MAX(c%02d) FROM episode GROUP BY idShow", VIDEODB_ID_EPISODE_RUNTIME);
```

Comment:
I'm confused as we choose to stay away from streamdetails previously in this PR and taking this data now seems weird. I would like to stick to the scraped values, even if streamdetails are already bleeding into them. Note, that tvdb doesn't know of the runtime of each episode of a multiepisode, but only the complete block.
I still think this is the way to go and would really dislike us splitting these values, as I watch on a file base, not on an episode base.